index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const path = require('path')
+const QRcode = require('qrcode')
+const bodyParser = require('body-parser')
+const knex = require('knex')({
+  client: 'pg',
+  connection: 'postgres://localhost:5432/qrcodes'
+})
+
+const app = express()
+
+app.use(express.static(path.join(__dirname, 'public')))
+app.use(bodyParser.json())
+
+app.post('/generate', (req, res) => {
+  QRcode.toDataURL(req.body.url, {scale: 25}, (err, url) => {
+    if (err) throw console.error(err)
+    const qrCode = req.body
+    qrCode.qr = url
+    const query = knex
+      .insert(req.body)
```

Comment:
insert here `qrCode`, and not the `req.body`, otherwise you're missing the generate qr code